Wrappers/Python/cil/framework/__init__.py

```diff
@@ -29,7 +29,7 @@
 from .framework import find_key, message
 from .framework import DataProcessor, Processor
 from .framework import AX, PixelByPixelDataProcessor, CastDataContainer
-from .BlockDataContainer import BlockDataContainer
-from .BlockGeometry import BlockGeometry
+from .BlockFramework import BlockDataContainer
+from .BlockFramework import BlockGeometry
```

Comment:
Technically this is a breaking change, right?

Do you think it's unlikely anyone would've done `from cil.framework.BlockGeometry import BlockGeometry` as the tests used to?

Reply:
This is a breaking change, but as @MargaretDuff says we hardly use it that way. Is there an alternative?

Reply:
Ok fine by me, but we shoyld rename `cil/framework/BlockFramework.py` to just `cil/framework/block.py` IMO